driver_scripts/driver_grid.orion.sh

```diff
@@ -83,19 +83,31 @@ export vegsoilt_frac='.false.'  # When true, outputs percent of each
                                # outputs the dominant category. A
                                # Fortran logical, so include the dots.
 
```

Comment:
These lines - <<< HEAD and >>> develop - are the result of a merge conflict. Please clean up.

Reply:
Done